src/components/card/card.scss

```diff
@@ -105,15 +105,19 @@
 
 @media only screen and (max-width: $desktop - 1) {
     .card {
-        .validation-message {
-            position: relative;
-            transform: none;
-            margin: inherit;
-            width: 100%;
-            height: inherit;
-
-            &:before {
-                display: none;
+        .form {
+            .row {
+                .validation-message {
+                    position: relative;
+                    transform: none;
+                    margin: inherit;
+                    width: 100%;
+                    height: inherit;
+
+                    &:before {
+                        display: none;
+                    }
```

Comment:
This actually is another example of why not to make it more specific unless you have to — then every time you want to change that class, you have to remember to repeat the same nesting as you have everywhere else in order for the overrides to work. If you make the original styles as generic as possible then you can override them in other places with just one level of nesting, like `.some-other-class .validation-message`, rather than `.some-other-class .form .row .validation-message`.